const axios = require('axios');module.exports = {  config: {    name: 'serenis',    version: '1.0',    author: 'SynthiaTech Innovations',    role: 0,    category: 'Ai-Chat',    shortDescription: { en: `Serenis is your companion for\n| tranquility, offering a calm and\n| empathetic presence. Experience a\n| soothing journey with Serenis as your\n| guide.` },    longDescription: { en: `Serenis is your companion for\n| tranquility, offering a calm and empathetic presence. Experience a\n| soothing journey with Serenis as your\n| guide.` },    guide: { en: '{pn}serenis [query]' },  },  onStart: async function ({ api, event }) {    try {      const query = args.join(" ");      if (query) {        const processingMessage = await api.sendMessage(`Asking 🌺 Serenis. Please wait a moment...`, event.threadID);        const response = await axios.get(`https://lianeapi.onrender.com/@synthiaTech/api/serenis?query=${encodeURIComponent(query)}`);                if (response.data && response.data.message) {          await api.sendMessage({ body: response.data.message.trim() }, event.threadID, event.messageID);          console.log(`Sent 🌺 Serenis's response to the user`);        } else {          throw new Error(`Invalid or missing response from 🌺 Serenis API`);        }        await api.unsendMessage(processingMessage.messageID);      }    } catch (error) {      console.error(`❌ | Failed to get 🌺 Serenis's response: ${error.message}`);      api.sendMessage(`❌ | An error occured. You can try typing your query again or resending it. There might be an issue with the server that's causing the problem, and it might resolve on retrying.`, event.threadID);    }  },};